### PR TITLE
Fix no-current of window source

### PR DESCRIPTION
**Pull request by thinca**
`:Unite window:all:no-current` excludes current window
form each tabpage.
It should exclude from only current tabpage.

### CODE REVIEW

File: autoload/unite/sources/window.vim [hunk 1]
```diff
@@ -39,13 +39,15 @@ let s:source = {
       \}
 
 function! s:source.hooks.on_init(args, context) "{{{
+  let no_current = index(a:args, 'no-current') >= 0
   if index(a:args, 'all') >= 0
     let a:context.source__candidates = []
+    let cur_tabnr = tabpagenr()
     for tabnr in range(1, tabpagenr('$'))
-      let a:context.source__candidates += s:get_windows(a:args, tabnr)
+      let a:context.source__candidates += s:get_windows(tabnr == cur_tabnr && no_current, tabnr)
```
Comment (Shougo) [inline autoload/unite/sources/window.vim]: This line is too long(90 column).
Can you insert a new line?

**Comment by Shougo** [comment]
Thank you!
It seems good.

File: autoload/unite/sources/window.vim (hunk 1 above)
Comment (thinca) [inline autoload/unite/sources/window.vim]: Fixed.

Comment (Shougo) [inline autoload/unite/sources/window.vim]: Thanks.

**Comment by Shougo** [comment]
Thanks!

**Pull request merged**